src/components/Expandable.stories.tsx

```diff
@@ -20,26 +23,68 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
-const Content = <>
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque ipsa
-voluptatibus soluta nobis unde quisquam temporibus magnam debitis quidem.
-Ducimus ratione corporis nesciunt earum vel est quaerat blanditiis dolore
-ipsa?&nbsp;
-<a href='/?path=/story/components-expandable--default'>Lorem link</a>
-</>
+const Content = (
+  <>
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Neque ipsa
+    voluptatibus soluta nobis unde quisquam temporibus magnam debitis quidem.
+    Ducimus ratione corporis nesciunt earum vel est quaerat blanditiis dolore
+    ipsa?&nbsp;
+    <a href='/?path=/story/components-expandable--default'>Lorem link</a>
+  </>
+);
 
 export const Default: Story = {
   args: {
     header: 'Expandable Header',
     children: Content
   }
-}
+};
 
 export const OpenOnLoad: Story = {
   args: {
     ...Default.args,
     header: 'Expandable Header',
     openOnLoad: true
   }
-}
+};
+
+export const TestExpandCollapse: Story = {
+  play: async ({ canvasElement }) => {
+    const waitTime = 500;
+    const canvas = within(canvasElement);
+
+    // Wait for initialization
+    await sleep(waitTime);
+
+    const heading = canvas.getByText(OpenOnLoad.args.header);
+    expect(heading).toBeInTheDocument();
+
+    // Content wrapper collapsed
+    const showHide = canvas.getByRole('button');
+    expect(showHide.getAttribute('aria-expanded')).toMatch('false');
+
+    // Content hidden
+    const content = canvas.queryByText(/lorem ipsum/gi);
+    expect(content).not.toBeVisible();
 
+    // Toggle
+    userEvent.click(showHide);
+    await sleep(waitTime);
+
+    // Content visible
+    expect(showHide.getAttribute('aria-expanded')).toMatch('true');
+    expect(content).toBeVisible();
+
+    // Toggle
+    userEvent.click(showHide);
+    await sleep(waitTime);
+
+    // Content hidden
+    expect(showHide.getAttribute('aria-expanded')).toMatch('false');
+    expect(content).not.toBeVisible();
+  },
+  args: {
+    ...Default.args,
+    header: 'Expandable Header'
+  }
+};
```

Comment:
This interaction test verifies the expand/collapse functionality that I failed to implement via RTL in `Expandable.test.tsx`.  That test (`Shows/Hides correctly`) is currently skipped but we can investigate further at a later date.